docs/community/README.md

```diff
@@ -0,0 +1,132 @@
+
+Knative is composed of many different components. We use GitHub Issues to track
+bug reports and feature requests. While it's most helpful if you know the
+specific component that the bug is happening in, we understand that it can be
+difficult to tell sometimes. A clear bug report in the wrong component is much
+better than a partial report in the right component, so feel free to file your issue in these three main repositories if you're not sure:
+
+* [Serving](https://github.com/knative/serving/issues/new?template=bug-report.md)
+* [Eventing](https://github.com/knative/eventing/issues/new?template=bug.md)
+* [Functions](https://github.com/knative/func/issues/new)
+
+A good bug report should include:
+
+- What you were trying to do, and what happened
+- What version of Knative and Kubernetes you are usinG (if using a cloud provider, indicate which one)
+- Relevant resource yaml, HTTP requests, or log lines
+
+## Community Meetups
+
+This virtual event is designed for end users, a space for our community to meet,
+get to know each other, and learn about uses and applications of Knative.
+
+Catch up with past community meetups on our [YouTube channel](https://www.youtube.com/playlist?list=PLQjzPfIiEQLLyCyLBKLlwDLfE_A-P7nyg).
+
+Stay tuned for new events by subscribing to the
+[calendar](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com&ctz=America%2FLos_Angeles) ([iCal export file](https://calendar.google.com/calendar/ical/knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com/public/basic.ics))
+and following us on [Twitter](https://twitter.com/KnativeProject).
+
+## Communication Channels
+
+Much of the community meets on [the Knative Slack](https://slack.knative.dev/).
+
+We also have user
+([`knative-users@googlegroups.com`](https://groups.google.com/forum/#!forum/knative-users))
+and developer
+([`knative-dev@googlegroups.com`](https://groups.google.com/forum/#!forum/knative-dev))
+mailing lists for discussions, and for access to documents in the shared Google
+Drive. Access to `knative-users@` is automatically approved; access to
+`knative-dev@` is handled via a lightweight approval process -- it helps if you
+provide a one-sentence description when requesting access.
+
+Feature design is generally done via Google Docs stored in a [shared Google
+Drive](https://drive.google.com/drive/folders/0AM-QGZJ-HUA8Uk9PVA). Due to
+limitations of shared drives, the default access control is that all documents
+are *readable* by `knative-users@` and *commentable* and *editable* by
+`knative-dev@`. Documents generally cannot be deleted without special
+intervention.
+
+We also use GitHub Issues and [GitHub
+projects](https://github.com/orgs/knative/projects) for tracking longer-term
+efforts, including each working group's roadmap, and the backlog for oversight
+committees like the Technical Oversight Committee and the Steering Committee.
```

Comment:
nit: should we move this to just after the slack bit? 

My thinking is that engagement usually goes something like  Slack -> Github -> mailing lists , so might be good to reflect that here

Reply:
Good thought, done.